charts/icm-as/values.yaml

```diff
@@ -28,8 +28,6 @@ image:
   pullPolicy: IfNotPresent
   # 'appServer' container provides the application / service endpoint
   repository: intershophub/icm-as:11.0.5
-  command: |
```

Comment:
If an `if`-condition would be used for the icm-as.image.command block this would be correct here.